README.md

```diff
@@ -56,25 +53,21 @@ instance-1  us-central1-a  n1-standard-1               10.240.0.2   8.34.219.29
 
 ### Installing additional components
 
-If you need to run additional gcloud components, you need to extend the base image:
+By default, all gcloud components are installed on the default image: 
 
-For example, for appengine-python:
-```dockerfile
-FROM google/cloud-sdk
-RUN gcloud components install app-engine-python
-```
+- [https://cloud.google.com/sdk/downloads#apt-get](https://cloud.google.com/sdk/downloads#apt-get)
+
+The debian-slim image (google/cloud-sdk-docker:alpine:159.0.0-slim), contains no additional components but you are welcome to extend the image or supply --build-args during the build state:
```

Comment:
`google/cloud-sdk-docker:alpine:159.0.0-slim` looks wrong.

Also we don't have `:debian-slim` image. It's just `:slim`.